legend-engine-xt-nonrelationalStore-mongodb-pure/src/main/resources/core_nonrelational_mongodb.definition.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name" : "core_nonrelational_mongodb",
+  "pattern" : "(meta::external::store::mongodb::metamodel|meta::external::store::mongodb::mapping|meta::protocols::pure|meta::pure::mapping|meta::pure::executionPlan|meta::pure::runtime|meta::pure::runtime::connection::authentication|meta::pure::alloy::connections|meta::pure::router|meta::external::language::java)(::.*)?",
```

Comment:
This is too wide.  All your packages should narrow down to mongo related paths.

Reply:
Removed it